src/config/mongoose_config_parser_toml.erl

```diff
@@ -410,7 +410,13 @@ pool_option([<<"call_timeout">>|_], V) -> [{call_timeout, V}].
 %% path: outgoing_pools.*.connection
 -spec connection_options(path(), toml_section()) -> [option()].
 connection_options([_, _, <<"rdbms">>|_] = Path, Options) ->
-    [{server, rdbms_server(Path, Options)}];
+    {Server, Interval} = rdbms_server(Path, Options),
+    case Interval of
```

Comment:
I'm having the feeling that this complicates the rbdms_server function, why don't we just extract the interval in place, like
```suggestion
    Server = rdbms_server(Path, Options),
    Interval = maps:get(<<"keepalive_interval">>, M, undefined),
    case Interval of
```